webrtc.html

```diff
@@ -8941,11 +8941,20 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           started, then abort these steps.</p>
         </li>
         <li>
-          <p>Attempt to send <var>data</var> on <var>channel</var>'s
-          <a>underlying data transport</a>; if the data cannot be sent, e.g.
-          because it would need to be buffered but the buffer is full, the user
-          agent MUST abruptly <a>close</a> <var>channel</var>'s <a>underlying
-          data transport</a> <a>with an error</a>.</p>
+          <p>If the size of <var>data</var> exceeds the value of <code><a
+          data-link-for="RTCSctpTransport">maxMessageSize</a></code> on
+          <var>channel</var>'s associated <code>RTCSctpTransport</code>,
+          <a>throw</a> an <code>InvalidAccessError</code>.</p>
```

Comment:
Perhaps we should go with an operation error here as well since InvalidAccessError is deprecated.